pina/problem/inverse_problem.py

```diff
@@ -1,7 +1,6 @@
 """Module for the ParametricProblem class"""
-
+import torch
```

Comment:
what's happening with inverse problems?

Reply:
Just need import torch for torch.rand at line 54